decidim-admin/app/controllers/decidim/admin/static_pages_controller.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all pages at the admin panel.
+    #
+    class StaticPagesController < ApplicationController
+      def index
+        authorize! :index, Decidim::StaticPage
```

Comment:
I think `Decidim` is redundant here. Applies to all other controllers as well (outside of the scope of this PR)